src/ISA/ISA/ArcTypes/ArcTable.fs

```diff
@@ -81,6 +81,15 @@ type ArcTable(name: string, headers: ResizeArray<CompositeHeader>, values: Syste
         fun (table:ArcTable) ->
             table.TryGetCellAt(column, row)
 
+    member this.IterColumns(mapping: CompositeColumn -> unit) =
```

Comment:
Really miniscule change, but in `Iter` the function is called `action`, not `mapping`